spanner/sessionclient.go

```diff
@@ -396,14 +412,30 @@ func (sc *sessionClient) sessionWithID(id string) (*session, error) {
 // optimal usage of server side caches.
 func (sc *sessionClient) nextClient() (spannerClient, error) {
 	var clientOpt option.ClientOption
+	var channelID uint64
 	if _, ok := sc.connPool.(*gmeWrapper); ok {
 		// Pass GCPMultiEndpoint as a pool.
 		clientOpt = gtransport.WithConnPool(sc.connPool)
```

Comment:
I don't know whether anything has been agreed upon regarding channel number when using GCPMultiEndpoint, but in the current implementation, the channel number will always be zero. Is that intentional?

Reply:
Nothing indeed has yet been agreed for GCPMultiEndpoint. I was thinking that that wouldn't be commonly used. Is it a show stopper for the initial code in?